packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -57,25 +57,45 @@ The new templating syntax is different from the EL expression-based Okta email t
+
+There are slight differences between EL templating syntax and VTL syntax. For example, the following EL and VTL syntax call the same function.
 
-You can now reference any attribute in a user's Okta User Profile in your email templates. The notation is `$user.profile.attributeName`. For example, to reference the displayName attribute of the user's Okta user profile, you would use `$user.profile.displayName`.
+EL templating syntax:
+
+`${f:formatTimeDiffHoursNowInUserLocale(org.activationTokenExpirationHours)}`
+
+VTL syntax:
+
+`$f.formatTimeDiffHoursNowInUserLocale($org.activationTokenExpirationHours)`
 
-See [Profile object](/docs/reference/api/users/#profile-object) for more information on the available user profile attributes.
+| Expression                                                    | Definition                                                                                                          |
+|---------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| replace(String input, String matchString, String replacement) | Replaces all occurrences of the match string with the replacement string                                           |
+| substringAfter(String input, String matchString)              | Returns the input substring after the occurrence of a given match string, or an empty string if no occurrence of the match string exists in the input string |
+| substringBefore(String input, String matchString)             | Returns the input substring before the occurrence of a given match string, or an empty string if no occurrence of the match string exists in the input string |
+| toLowerCase(String input)                                     | Converts the given input string to all lowercase                                                                   |
+| toUpperCase(String input)                                     | Converts the given input string to all uppercase                                                                   |
+| substring(String input, int startIndex, int endIndex)         | Extracts a range of characters from the given input string                                                         |
+| formatTimeDiffHoursNow(int hours)                             | Produces a formatted duration string from the current time to the given number of hours                                                 |
+| formatTimeDiffHoursNowInUserLocale(int hours)                 | Produces a localized formatted duration string for the given number of hours                                       |
+| formatTimeDiffDateNow(Date date)                              | Produces a formatted duration string for the given date                                                            |
```

Comment:
Just Identified a bug for this function call with Date param. This does not work as expected. cc @vigneshnarasimankumar-okta